python/functions.py

```
@@ -209,16 +209,13 @@ def filter_people_by_age(people: List[Person], min_age: int) -> List[Person]:
 def get_person_names(people: List[Person]) -> List[str]:
     return [p.name for p in people]
 
-# Exception handling
+# Exception handling demonstration
 def demonstrate_exception_handling() -> None:
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)